### PR TITLE
Patch fix for #288

### PR DESCRIPTION
**Pull request by Skeletrox**
- Remove unnecessary UTC function call
- Change path of the called script
- Fix logical fallacy

### CODE REVIEW

File: devmgmtV2/telemetry_cron.js [hunk 1]
```diff
@@ -135,15 +135,16 @@ let repeatedlyCheckForInternet = () => {
...
+    const cmd = '/opt/opencdn/CDN/getall_stations.sh';
     exec(cmd, { shell: '/bin/bash' }, (err, stdout, stderr) => {
         // console.log('Error: ' + err);
-        // console.log('stdout: ' + stdout);
+        console.log('stdout: ' + stdout);
         // console.log('stderr: ' + stderr);
         // console.log("Checking user status...");
-        if (err !== null) {
-            let usersConnected = stdout.trim();
+        if (err === null) {
```
Comment (rishabh1403) [inline devmgmtV2/telemetry_cron.js]: if (!err)

File: devmgmtV2/telemetry_cron.js [hunk 2]
```diff
@@ -110,7 +110,7 @@ let generateTelemetry = (telemetryType, telemetryValue) => {
     }
     if (telemetryNow !== null) {
```
Comment (rishabh1403) [inline devmgmtV2/telemetry_cron.js]: if (!telemetryNow)

Comment (Skeletrox) [reply devmgmtV2/telemetry_cron.js]: Wouldn't that negate the logic, causing the telemetry to be "generated" when it is neither pertaining to users connected or internet connectivity?

Comment (rishabh1403) [reply devmgmtV2/telemetry_cron.js]: if(telemetry)

**Pull request merged**